packages/webapp/src/main/styles.css

```diff
@@ -70,16 +91,16 @@ body {
   cursor: pointer;
 }
 
-.theme-toggle.dark .sun-and-moon > .sun-beams {
+.theme-toggle.dark [class$='sun-and-moon'] > [class$='sun-beams'] {
```

Comment:
That looks weird. Why can't we keep the normal class selector?

Reply:
"svgr webpack" package added prefix "tooltip_svg__" to the classname that I assigned. 
Since I did not have control over it and was not sure that's the constant prefix that will be added, I thought of using the selector that selects the class ending with the class name that I assigned. 

![image](https://user-images.githubusercontent.com/14681902/169868399-422e1b7c-5811-434f-a4de-30d555043637.png)